redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/request/DatabaseV4BuildRequest.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.database.request;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.JsonEntity;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.requests.DatabaseV4Request;
+import com.sequenceiq.cloudbreak.doc.ModelDescriptions.Database;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DatabaseV4BuildRequest implements JsonEntity {
+
+    @NotNull
+    @ApiModelProperty(value = Database.DATABASE_REQUEST, required = true)
+    private DatabaseV4Request rdsConfigRequest;
```

Comment:
Rename to databaseRequest (along with accessor methods)

Reply:
done, thanks.

Reply:
Hm, this file is deleted later on :)